compiler/test/dotty/tools/vulpix/TestConfiguration.scala

```diff
@@ -47,7 +47,7 @@ object TestConfiguration {
   val yCheckOptions = Array("-Ycheck:all")
 
   val commonOptions = checkOptions ++ noCheckOptions ++ yCheckOptions
-  val defaultOptions = TestFlags(basicClasspath, commonOptions)
+  val defaultOptions = TestFlags(basicClasspath, commonOptions) and "-Ykind-polymorphism" // FIXME remove "-Ykind-polymorphism"
```

Comment:
The main blocker for this PR is that any code that uses `scala.quote.Type` requires `-Ykind-polymorphism`

Reply:
As discussed during today's meeting kind polymorphism is mature enough to be enabled by default. Though I decided to keep a `-Yno-kind-polymorphism` to disable it if needed.